src/transaction/log_replication_atomic.cpp

```diff
@@ -239,8 +239,15 @@ namespace cublog
 	    const LOG_REC_SCHEMA_MODIFICATION_LOCK log_rec =
 		    m_redo_context.m_reader.reinterpret_copy_and_add_align<LOG_REC_SCHEMA_MODIFICATION_LOCK> ();
 
+	    // check if m_locked_classes[header.trid] has log_rec.classoid
```

Comment:
Nit. I think it's redundant and even reveals details unnecessarily.
```suggestion
```

Reply:
OK, I will remove it